src/styles.css

```
@@ -865,4 +901,5 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
   box-shadow: 0 4px 24px rgba(0, 0, 0, 0.18);
   transition: opacity 0.2s ease, transform 0.2s ease;
   pointer-events: none;
+ main
```

Stylelint (17.13.0)

[error] Unknown word main (CssSyntaxError)

(CssSyntaxError)